### PR TITLE
Fix legacy detection when the morph has no trunk

### PR DESCRIPTION
**Pull request by wizmer**
The finding of oblique roots needed by the legacy detection was failing for
morphologies that have no trunk but have some obliques nonetheless.

### CODE REVIEW

File: tests/test_repair.py [hunk 1]
```diff
@@ -331,6 +331,14 @@ def test_repair_no_intact_axon():
         test_module.repair(filename, outfilename, axons=[filename])
 
 
+def test_repair_no_trunk():
+    '''Test repair when the morph has oblique sections but no trunk'''
+    filename = DATA_PATH / 'Fluo42_right.h5'
+    with TemporaryDirectory('test-no-trunk') as tmp_folder:
+        outfilename = Path(tmp_folder, 'out.asc')
+        test_module.repair(filename, outfilename, legacy_detection=True)
```
Comment (asanin-epfl) [inline tests/test_repair.py:339]: don't we need to assert here the number of expected obliques?

Comment (wizmer) [reply tests/test_repair.py:339]: Do you mean before calling `repair()` ?

Comment (asanin-epfl) [reply tests/test_repair.py:339]: I mean after. Just as in `test_repair_axon`. How can we verify that the previously missed obliques were counted this time?

Comment (wizmer) [reply tests/test_repair.py:339]: Ah, no they were not simply missed. The code was crashing ! :)
So just the fact that the code runs is enough to cover my change

Comment (asanin-epfl) [reply tests/test_repair.py:339]: ah, ok then.

**Review by asanin-epfl — APPROVED** [review]

**Comment by wizmer** [comment]
Thanks

**Pull request merged**